modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/objectDefinitionUtil.ts

```diff
@@ -130,9 +130,21 @@ export async function deleteRelationship(id: number) {
 		});
 	}
 	catch (error) {
-		Liferay.Util.openToast({
-			message: (error as Error).message,
-			type: 'danger',
+		const errorMessage = (error as Error).message;
+		Liferay.Util.openModal({
```

Comment:
@larissacribeiro Please add a blank line between the declaration of `errorMessage` and its usage.

Reply:
done!